src/styles/base/card.scss

```diff
@@ -1,17 +1,40 @@
 .card {
     background-color: $color-gray-light;
-    padding: 30px;
+    padding: 10%;
     border-radius: 5px;
     text-align: center;
+    margin-top: 30px;
+    border: none;
 
     &__img {
-        width: 100%;
+        width: 40%;
         border-radius: 50%;
-        margin: 20px 0;
+        margin: auto;
+        margin-top: 40px;
     }
 
     &__btn {
         display: flex;
         justify-content: center;
+        margin-top: 40px;
+
+        &--text {
```

Comment:
`text` and `icon` are *not* modifiers. They are elements.

**Use `.card__text` and `.card__icon` instead.**

```diff
- &--text {}
- &--icon {}
+ &__text {}
+ &__icon {}
```

> You will need to change your HTML to use new classes.